modules/ROOT/pages/stats-monitoring.adoc

```diff
@@ -208,6 +208,14 @@ This metric can be used to calculate the latency of `_changes` request.
 
 `_changes` request latency = `request_changes_time` / `request_changes_count`
 
+===== rev_send_latency
+
+Time between receiving the `/_changes` response to sending a revision.
```

Comment:
@adamcfraser @bbrks I copied this definition from [here](https://docs.google.com/spreadsheets/d/1aZjCIwwZYrUSaQCK63QUg7_jrbo0jJcsxPoXG2Q4s0o/edit#gid=0). I don't understand it. Should it read the following instead?
```suggestion
Time between receiving the `/_changes` request to sending back a revision.
```

Reply:
The`/_changes` request originates from SG, and the response back from the client contains a list of revisions that it wants to receive.

This stat is measuring the time between the client asking for some revisions via that response, and us sending the actual revision(s) to it.

Hopefully that clarifies.

Reply:
@bbrks Yes definitely, including both paragraphs.

```suggestion
In a pull replication, Sync Gateway sends a `/_changes` request to the client.
The client responds with the list of revisions that it wants to receive.

`rev_send_latency` is measuring the time between the client asking for some revisions via the `/_changes` response, and Sync Gateway sending that revision to the client.
```